packages/website/ts/components/inputs/token_amount_input.tsx

```diff
@@ -64,8 +72,9 @@ export class TokenAmountInput extends React.Component<TokenAmountInputProps, Tok
             ? ZeroEx.toUnitAmount(this.props.amount, this.props.token.decimals)
             : undefined;
         const hasLabel = !_.isUndefined(this.props.label);
+        const style = !_.isUndefined(this.props.style) ? this.props.style : { height: hasLabel ? 84 : 62 };
```

Comment:
Let's make `84` and `62` into constants above with good names `HEIGHT_WITH_LABEL` and `HEIGHT_WITHOUT_LABEL`